src/Symfony/Bundle/WebProfilerBundle/README.md

```diff
@@ -1,6 +1,12 @@
 WebProfilerBundle
 =================
 
+The Web profiler bundle is a **development tool** that gives detailed
+information about the execution of any request.
+
+**Never** enable it in production environments as it will lead to major security
```

Comment:
"Production environment" is ambiguous. Is it the Symfony production environment (`prod`), or does it represent the production environment of your server?

In case of the latter, it should only be made accessible if really needed, in `dev` and well secured.

Reply:
Here, we are talking about production servers. I will update the comment to make it clear.

Reply:
done

Reply:
That's better! 👍